app/src/main/java/com/github/se/cyrcle/io/PreferenceStorage.kt

```diff
@@ -0,0 +1,41 @@
+package com.github.se.cyrcle.io
+
+import android.content.Context
+import androidx.datastore.core.DataStore
+import androidx.datastore.preferences.core.Preferences
+import androidx.datastore.preferences.core.edit
+import androidx.datastore.preferences.preferencesDataStore
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.map
+
+val Context.preferencesDataStore: DataStore<Preferences> by preferencesDataStore(name = "settings")
```

Comment:
I think the definition of a "preference in the comments of this class could be good. I get what it is, but I feel like it could be made clearer in your code and in your PR for the rest of the group who is not reviewing you :)

Reply:
Indeed, some documentation would be require for this, I'll be on it !

Reply:
Thanks!